test/functional/bipdersig.py

```diff
@@ -12,12 +12,10 @@ def __init__(self):
         super().__init__()
         self.num_nodes = 3
         self.setup_clean_chain = False
+        self.extra_args = [["-blockversion=2"], ["-blockversion=3"]]
```

Comment:
Shouldn't this have an empty list first? Node 0 has no opts, Node 1 has blockversion=2, etc?